src/index.js

```diff
@@ -5,12 +5,39 @@ import UrlPattern from 'url-pattern';
 
 const DEFAULT_REQUEST_METHOD = 'GET';
 
+const fetchFactoryTemplates = {
+  findOne: {
+    method: 'GET',
+  },
+  findAll: {
+    method: 'GET',
+  },
+  create: {
+    method: 'POST',
+  },
+  update: {
+    method: 'PUT',
+  },
+};
+
 const fetchFactory = {
-  create(options, methods) {
+  create(options, methods = {}) {
     this.factory = {};
 
     this.defaultOptions = options;
 
+    const templateMethods = (options.methods || []).map((methodName) => {
```

Comment:
would you omit the `methods` argument passed to `fetchFactory.create` if you wanted to create a fetchFactory without any of the template methods? If yes, this would blow up because the expression `(options.methods || [])` returns an empty object, which doesn't have a `map` method.

It could be caught by setting the default to `undefined` and write a test where `create` is called with no `methods` argument?